widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -298,151 +319,138 @@ export function ConfirmWalletsModal(props: PropTypes) {
-                    )}
-                  />
-                </ListContainer>
-                {index !== requiredWallets.length - 1 && <Divider size={32} />}
-                {index === requiredWallets.length - 1 &&
-                  customDestinationEnabled && (
-                    <>
-                      <CustomDestination>
-                        <CollapsibleRoot
-                          selected={showCustomDestination}
-                          open={showCustomDestination}
-                          onOpenChange={(checked) => {
-                            if (!checked) {
-                              resetCustomDestination();
-                            } else {
-                              if (
-                                !isWalletRequiredFor(
-                                  lastStepToBlockchain?.name ?? ''
-                                )
-                              ) {
-                                setSelectableWallets((selectableWallets) =>
-                                  selectableWallets.map((selectableWallet) => {
-                                    if (
-                                      selectableWallet.chain ===
-                                      lastStepToBlockchain?.name
-                                    ) {
-                                      return {
-                                        ...selectableWallet,
-                                        selected: false,
-                                      };
-                                    }
-                                    return selectableWallet;
-                                  })
-                                );
+              const key = `wallet-${index}`;
+              return (
+                <div key={key}>
+                  <Title>
+                    <Typography
+                      variant="title"
+                      size="xmedium">{`Your ${blockchain?.displayName} wallets`}</Typography>
+                    <Typography
+                      variant="label"
+                      color="$neutral900"
+                      size="medium">{`You need to connect a ${blockchain?.displayName} wallet.`}</Typography>
+                  </Title>
+                  <Divider size={24} />
+                  <ListContainer>
+                    <WalletList
+                      chain={requiredWallet}
+                      isSelected={isSelected}
+                      selectWallet={onChange}
+                      multiWallets={config?.multiWallets ?? true}
+                      supportedWallets={config?.wallets ?? []}
+                      config={config}
+                      limit={NUMBER_OF_WALLETS_TO_DISPLAY}
+                      onShowMore={setShowMoreWalletFor.bind(
+                        null,
+                        blockchain?.name ?? ''
+                      )}
+                    />
+                  </ListContainer>
+                  {index !== requiredWallets.length - 1 && (
```

Comment:
`const isLastWallet = index !== requiredWallets.length - 1` and use it here and line 359.